site/docs/foundations/data-visualization/chart-anatomy.mdx

```diff
@@ -0,0 +1,109 @@
+
+<Image
+  src="/img/foundations/dataviz-labels.png"
+  srcDark="/img/foundations/dataviz-labels-dark.png"
+  alt="Diagram of labels on a bar chart."
+/>
+
+### Axes title
+
+The axis title should summarize the variables shown. Consider including the unit of measurement in the title to avoid repeating it on the axis labels, e.g., M for millions, or for percentages.
+
+### Axes labels
+
+Label each axis clearly and concisely to reduce visual clutter. Simplify axis labels by including the unit of measurement in the axis title. Keep labels horizontal where possible to maintain readability.
+
+## Grid lines
+
+Grid lines are vertical or horizontal lines that divide a chart, helping users read data values and make visual comparisons.
+
+When possible, keep the number of grid lines to a minimum to prevent your chart becoming cluttered. Some charts require both horizontal and vertical grid lines, i.e. bubble and scatter chart. This is built into the Salt chart package and doesn’t require manual adjustment.
+
+<Image
+  src="/img/foundations/dataviz-grid.png"
+  srcDark="/img/foundations/dataviz-grid-dark.png"
+  alt="Diagram of grid lines used in a chart."
+/>
+
+## Legends
+
+### Usage
+
+The legend uses symbols and labels to identify each data series and its corresponding segment within the chart. In Salt, the legend is an opt-in feature.
+
+Certain chart types, such as scatter and bubble charts, use legend symbols that visually match the representation of the data points within the chart. This is a design choice that enhances clarity and makes the chart easier to interpret.
+
+<Image
+  src="/img/foundations/dataviz-legend.png"
+  srcDark="/img/foundations/dataviz-legend-dark.png"
+  alt="Diagram of different legend types."
+/>
+
+### Legend layouts
+
+Position your legend where it’s most appropriate in your layout, considering the number of legend items. In Salt, chart legends are preset to be vertical and right-aligned. Use a vertical layout when displaying many legend items, if space allows.
+
+<Image
+  src="/img/foundations/dataviz-legend-layouts.png"
+  srcDark="/img/foundations/dataviz-legend-layouts-dark.png"
+  alt="Diagram of different legend orientations."
+/>
+
+### Pagination
+
+When you have more legend items than available to fit in the height of the chart, default pagination will be used. This is should not be confused with the Salt Pagination component.
+
+## User interactions
+
+Salt charts include interactions that help users focus on or reveal additional information about your chart data, for example:
+
+- Tooltips that appear on hover
+- Individual data series or points that users can show, hide or highlight
+
+### Selected state
+
+Users can select or click on a legend item to show or hide the related data series, isolating data points that are important to them.
+
+<Image
```

Comment:
Is this reflective of the interaction? I can see the opacity change in the legend but not in the chart.

Reply:
I can't see the screenshot, would you be able to share it on Teams?